### ISSUE TITLE
DetectLandmarksInVideo() doesn't care about validation boundary and returns true

### ISSUE THREAD

**Issue by ChrizzyM87** [TadasBaltrusaitis/OpenFace#568]
Dear Tadas,
i noticed a behaviour of DetectLandmarksInVideo() that really confuses me. The function returns true, but the detection_success flag of the model is false and the detection certainty lies under the validation boundary.
Obviously this is due to the return statement in line [375](https://github.com/TadasBaltrusaitis/OpenFace/blob/master/lib/local/LandmarkDetector/src/LandmarkDetectorFunc.cpp#L375) of the LandmarkDetectorFunc.cpp. You wrote: 
> do not care if it was successful as the validator might be wrong, so continue trackig

But I dont get, why this should make sense here. In my application it leads to a behaviour that is not intended: e.G. If I got no face in the video, every time the model is reinitialized and the face detector found some garbage, my detection success flag gets true (I take the return value, like you do in your examples), but in the next frame of course it gets changed to false, because now the validation boundary is effective.
Of course I can bypass the problem, but maybe it would be reasonable to change the code here!?
Thanks in advance!
Best greetings from Leipzig, Christoph

**Comment by TadasBaltrusaitis** [comment]
I agree that the behavior is a bit strange and confusing, the original intention of this code was because the validator was not as reliable as the face detector and the idea was to trust the detector more, also in some cases if the detector initialized a bit further from the face the tracker had an extra frame to get closer to the correct location and the validator would return a value above the threshold. 

But as you mention it leads to weird behavior with false positives from face detector. I'll run my test suite with your suggested change and see if that leads to similar results.
